engine/api/environment/environment.go

```diff
@@ -473,6 +473,39 @@ func Permission(key string, envName string, u *sdk.User) int {
 	return u.Permissions.EnvironmentsPerm[sdk.UserPermissionKey(key, envName)]
 }
 
+// CountInValueVarData represents the result of CountInVarValue function
+type CountInValueVarData struct {
+	Name  string
+	Count int64
+}
+
+// CountInVarValue counts how many time a pattern is in variable value for the given project
+func CountInVarValue(db gorp.SqlExecutor, key string, value string) ([]CountInValueVarData, error) {
+	query := `
+		SELECT count(*), environment.name
+		FROM environment_variable
+		JOIN environment ON environment.id = environment_variable.environment_id
+		JOIN project ON project.id = environment.project_id
+		WHERE value like $2 AND project.projectkey = $1
```

Comment:
instead of `like`did you try postgres fulltext search ?
https://www.postgresql.org/docs/9.4/static/textsearch-intro.html#TEXTSEARCH-MATCHING

Reply:
there is other way to use tsvector: https://www.postgresql.org/docs/9.4/static/datatype-textsearch.html